x-pack/plugins/observability_solution/entity_manager/server/plugin.ts

```diff
@@ -73,6 +75,12 @@ export class EntityManagerServerPlugin
       router,
       logger: this.logger,
       server: this.server,
+      getClient: async ({ request }: { request: KibanaRequest }) => {
```

Comment:
Maybe call this `scopedClient`? I think that is the common pattern? To clarify that it's bound to a request